### ISSUE TITLE
`scss/operator-no-unspaced` false reports

### ISSUE THREAD

**Issue by ArmorDarks** [kristerkari/stylelint-scss#144]
Hi!

Following rule

```yaml
  scss/operator-no-unspaced: true
```

wrongly reports occurrences of "operators"-like characters in Sass map and extends:

```scss
$test: (
 \+:   10px,
 \+\+: 20px,
);

.test {
  @extend .h-trans--all\+;
}
```

```shell
source/styles/generic/_code.scss
  2:3   ×  Expected single space before "+"                        scss/operator-no-unspaced
  2:3   ×  Expected single space after "+"                         scss/operator-no-unspaced
  3:3   ×  Expected single space before "+"                        scss/operator-no-unspaced
  3:3   ×  Expected single space after "+"                         scss/operator-no-unspaced
  3:5   ×  Expected single space before "+"                        scss/operator-no-unspaced
  3:5   ×  Expected single space after "+"                         scss/operator-no-unspaced
  7:25  ×  Expected single space before "+"                        scss/operator-no-unspaced
  7:25  ×  Expected single space after "+"                         scss/operator-no-unspaced
```

![sublime_text_2017-07-18_19-36-13](https://user-images.githubusercontent.com/4460311/28328809-63223aaa-6bf0-11e7-836e-642c41eea437.png)

**Comment by kristerkari** [comment]
Thanks for reporting this! I guess the rule needs to just ignore the characters if they're not real operators.

**Comment by kristerkari** [comment]
Fixed in #148

**Comment by ArmorDarks** [comment]
Thanks! Tested, works!
